slices/tar.yaml

```diff
@@ -0,0 +1,31 @@
+package: tar
+
+essential:
+  - tar_copyright
+
+slices:
+  bins:
```

Comment:
Let's have `bins` contain all the binaries. We can have individual slices `rmt`, `tar`, `cat` etc. and then include all of those in `bins`.

Reply:
That would diverge from the existing SDF in 24.04 that this is a forward-port of?